### PR TITLE
Update to Outdated files in dev-1.17-ko.4 branch.

### PR DESCRIPTION
**Pull request by ysyukr**
### 36 out of 40 files to be modified

- [x] content/ko/docs/concepts/architecture/cloud-controller.md
- [x] content/ko/docs/concepts/containers/images.md
- [x] content/ko/docs/concepts/containers/runtime-class.md
- [x] content/ko/docs/concepts/overview/components.md
- [x] content/ko/docs/concepts/overview/kubernetes-api.md
- [ ] content/ko/docs/concepts/overview/working-with-objects/field-selectors.md
> 번역시 의미가 동일하여 미반영
- [x] content/ko/docs/concepts/services-networking/connect-applications-service.md
- [x] content/ko/docs/concepts/services-networking/endpoint-slices.md
- [x] content/ko/docs/concepts/services-networking/ingress-controllers.md
- [x] content/ko/docs/concepts/services-networking/service.md
- [x] content/ko/docs/concepts/workloads/controllers/replicaset.md
- [x] content/ko/docs/concepts/workloads/pods/ephemeral-containers.md
- [x] content/ko/docs/concepts/workloads/pods/pod-lifecycle.md
- [x] content/ko/docs/contribute/style/write-new-topic.md
- [x] content/ko/docs/reference/glossary/cluster.md
- [x] content/ko/docs/reference/glossary/container-runtime.md
- [x] content/ko/docs/reference/glossary/kube-controller-manager.md
- [x] content/ko/docs/reference/glossary/kube-scheduler.md
- [x] content/ko/docs/reference/glossary/pod-lifecycle.md
- [ ] content/ko/docs/reference/glossary/service-account.md
> namespace의 중복단어 삭제이지만, 한글 문서는 중복이 없어 미반영
- [x] content/ko/docs/reference/issues-security/security.md
- [x] content/ko/docs/reference/kubectl/cheatsheet.md
- [x] content/ko/docs/setup/_index.md
- [ ] content/ko/docs/setup/best-practices/certificates.md
> #18709 에서 반영됨
- [x] content/ko/docs/setup/best-practices/node-conformance.md
- [x] content/ko/docs/setup/learning-environment/minikube.md
- [x] content/ko/docs/setup/production-environment/container-runtimes.md
- [x] content/ko/docs/setup/production-environment/tools/kops.md
- [ ] content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md
> #18272 에서 반영됨
- [x] content/ko/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md
- [x] content/ko/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/declarative-config.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/imperative-config.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md
- [x] content/ko/docs/tutorials/hello-minikube.md
- [x] content/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html
- [x] content/ko/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md
- [x] content/ko/examples/minikube/Dockerfile
- [x] content/ko/includes/task-tutorial-prereqs.md

from #18917
/language ko

### CODE REVIEW

File: content/ko/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ card:
 
 {{% capture overview %}}
 쿠버네티스를 배포하면 클러스터를 얻는다.
-{{< glossary_definition term_id="cluster" length="all" prepend="클러스터는">}}
+{{< glossary_definition term_id="cluster" length="all" prepend="쿠버네티스 클러스터는 다음으로 구성된다">}}
```
[the remark this answers is not in the document]
Comment (ysyukr) [reply content/ko/docs/concepts/overview/components.md]: 이 부분은 원문 수식어가 `A cluster is` 에서 `A Kubernetes cluster consists of` 로 변경되었기에 반영하면서 어색해진듯 합니다.

그리고 실제 반영되었을 때 클러스터 용어에 대한 디테일한 설명이 다 나오는 것으로 보아 `쿠버네티스 클러스터는` 으로만 변경하는게 조금 더 자연스러울 것 같습니다.

**Comment by ysyukr** [comment]
@seokho-son 리뷰 감사합니다. 확인해서 업데이트했습니다 :)

**Comment by seokho-son** [comment]
감사합니다^^
/lgtm
/approve

**Pull request merged**